### PR TITLE
[NUI] Publish LottieAnimationView : TCSACR-293

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Publish LottieAnimationView : TCSACR-293

### API Changes ###
http://suprem.sec.samsung.net/jira/browse/TCSACR-293

### CODE REVIEW

**Comment by dongsug-song** [comment]
@hyunjushin @heeyongsong 
Hello,
please review this PR and check the ACR.
http://suprem.sec.samsung.net/jira/browse/TCSACR-293
thank you.

**Comment by dongsug-song** [comment]
no more review done, ACR accepted, so merged

File: src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs [hunk 1]
```diff
@@ -326,18 +358,17 @@ public int LoopCount
         }
 
         /// <summary>
-        /// Stop Behavior (Stop End Action)
+        /// Set or Get stop behavior.
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs:361]:
```suggestion
        /// Sets or gets the stop behavior.
```

File: src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs [hunk 2]
```diff
@@ -373,15 +404,14 @@ public StopBehaviorType StopBehavior
 
         #region Method
         /// <summary>
-        /// SetMinMaxFrame(int startFrame, int endFrame)
+        /// Set minimum and maximum frame.
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs:407]:
```suggestion
        /// Set the minimum and the maximum frame.
```

File: src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs [hunk 3]
```diff
@@ -394,57 +424,54 @@ public void SetMinMaxFrame(int minFrame, int maxFrame)
...
+            tlog.Fatal(tag, $"<[{GetId()}] Stop()");
             debugPrint();
             base.Stop();
-            tlog.Fatal(tag,  $"[{GetId()}]>");
+            tlog.Fatal(tag, $"[{GetId()}]>");
         }
 
         /// <summary>
-        /// GetContentInfo()
+        /// Get the list of layers' information (start frame, end frame) in Lottie file.
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs:467]:
```suggestion
        /// Get the list of layers' information such as the start frame and the end frame in the Lottie file.
```

**Comment by dongsug-song** [comment]
@journey2w 
Hello, I upload the patch;  https://github.com/Samsung/TizenFX/pull/1737 
thank you.

**Pull request merged**